.github/workflows/ci-go.yml

```diff
@@ -50,3 +50,33 @@ jobs:
           go get .
           gosec -exclude=G204,G304,G107 -exclude-dir=protobuf/ ./...  
         working-directory: src
+
+  go-ginkgo-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: v1.18
+        
+      - name: Setup Enviroment
+        id: vars
+        run: |
+          if [ ${{ github.ref }} == "refs/heads/dev" ]; then
```

Comment:
i don't think we need `tag` variable to be used anywhere in this workflow.

Reply:
resolved